src/messages/index.js

```diff
@@ -1,25 +1,25 @@
 import "./messages.css";
 
-const Messages = () => (
-  <div class="messages">
-    <div id="streak-message" class="show">
-      Current Streak: <span id="streak">@Model.Streak</span>
-    </div>
-    <div id="attempts-message">
-      Attempts Remaining Before Streak Lost: <span id="attempts-left">3</span>
-    </div>
-    <div id="incorrect-display">
-      <div class="incorrect-message">
-        Incorrect!
-        <span id="incorrect-message-reset">Streak Reset.</span>
-      </div>
-      <div class="incorrect-answer">
-        <a href="#answer" id="incorrect-answer-link">
-          Show Answer (Lose Streak)
-        </a>
-      </div>
-    </div>
-  </div>
-);
+const Messages = (props) => {
+  let visible = false;
+  let isCorrect = false;
+
+  if (props.checkWin !== 0) {
+    visible = true;
+  }
+  if (props.checkWin === 2) {
+    isCorrect = true;
+  }
+
+  if (visible) {
+    if (isCorrect) {
+      return <br />;
+    } else {
+      return <div className="incorrect-message">incorrect</div>;
+    }
+  } else {
+    return <div></div>;
```

Comment:
If you want to display nothing from a component, instead of doing an empty `<div>` you can just return `null`